Makefile

```diff
@@ -1,24 +1,25 @@
 S3_CP_ARGS=aws s3 cp --acl public-read
+RELEASE_VERSION?=
+
+S3_BUCKET_PATH=s3://observe/cloudformation/
+S3_EXISTING_VERSIONS=$(shell aws s3 ls $(S3_BUCKET_PATH) | grep $(RELEASE_VERSION) | wc -l)
```

Comment:
Calling `ListBucket` is expensive, use `HeadObject` to check for existence:

https://docs.aws.amazon.com/cli/latest/reference/s3api/head-object.html

If this exits 0, it exists. If it exits 254, that indicates an API error. Any other exit code is some other fatal error earlier than that, e.g., no credentials.

Reply:
resolved in [e93dc8f](https://github.com/observeinc/cloudformation-aws-collection/pull/35/commits/e93dc8f8fbfd74a79452f2fa9adadc7dadd41c64)